conbench/api/commits.py

```diff
@@ -18,7 +20,11 @@ def get(self):
         tags:
           - Commits
         """
-        commits = Commit.all(order_by=Commit.timestamp.desc(), limit=500)
+        if sha_arg := f.request.args.get("sha"):
+            shas = sha_arg.split(",")
+            commits = Commit.search([Commit.sha.in_(shas)])
+        else:
+            commits = Commit.all(order_by=Commit.timestamp.desc(), limit=500)
```

Comment:
The answer might be no, but is it possible to generalize this pattern that I see in a few places here?